python/core/auto_generated/pointcloud/qgspointcloudrendererregistry.sip.in

```diff
@@ -153,13 +153,18 @@ renderer was not found in the registry.
 Returns a list of available renderers.
 %End
 
-    static QgsPointCloudRenderer *defaultRenderer( const QgsPointCloudDataProvider *provider ) /Factory/;
+    static QgsPointCloudRenderer *defaultRenderer( const QgsPointCloudLayer *layer ) /Factory/;
 %Docstring
```

Comment:
This is an API break -- can you add a new method instead?

Reply:
This change was actually in #48321
It will be addressed in an upcoming PR along with some other fixes.